test_type_defs.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -o errexit
+
+npm install
+cd type-definitions/tests
+../../node_modules/flow-bin/cli.js
```

Comment:
also - when called from the context of npm scripts, node modules are added to PATH, so you could just write `flow` there.